core/domain/user_query_services.py

```diff
@@ -28,6 +28,24 @@ def save_new_query_model(
         has_not_logged_in_for_n_days=None, created_at_least_n_exps=None,
         created_fewer_than_n_exps=None, edited_at_least_n_exps=None,
         edited_fewer_than_n_exps=None):
+    """Saves a new UserQueryModel instance in user_models.
+
+    Args:
+        submitter_id: str. ID of the UserQueryModel instance.
+        inactive_in_last_n_days: int. Number of days user is inactive.
+        has_not_logged_in_for_n_days: int. Number of days user hasn't logged in.
+        created_at_least_n_exps: int. Lower bound for number of explorations
+        created by user.
```

Comment:
Indent continuation line by 4 spaces. It'll be more clear to write 'Minimum number of ...'

Reply:
Thanks, @vibhor98  :)  So, you are asking to replace `number of` with `minimum number of`?
Also, do I have to indent the last line above with 4 spaces?

Reply:
No, I mean it is clearer to write- 'Minimum number of ...' instead of 'Lower bound...' For indentation, indent continuation lines like- 38 by 4 spaces.